src/rateEngine/__tests__/LoadProfileScaler.test.ts

```diff
@@ -58,4 +58,20 @@ describe('LoadProfileScaler', () => {
       })
     });
   });
+
+  describe('toMonthlyKwh', () => {
+    it('returns a LoadProfile scaled to the monthly load', () => {
+      const monthlyLoad = times(12, (i) => ((i + 1) * 100));
+      const scaledLoadProfile = loadProfileScaler.toMonthlyKwh(monthlyLoad);
+
+      scaledLoadProfile.sumByMonth().forEach((monthlyTotal, idx) => {
+        expect(monthlyTotal).toBeCloseTo(monthlyLoad[idx]);
+      })
+    });
+
+    it('throws an error with an array without 12 elements', () => {
```

Comment:
i added an explicit check for array length here because i suspect that Future Me will very possibly get the input wrong at some point and TS isn't of use at runtime